jobs/configure-eirini/templates/eirini-daemonset.yaml.erb

```diff
@@ -107,15 +87,12 @@ spec:
       - name: varlibdockercontainers
         hostPath:
           path: /var/lib/docker/containers
-      - name: varvcapstore
-        hostPath:
-          path: /var/vcap/store/
-      - name: varvcapdata
+      - name: varvcapstoredockercontainers
         hostPath:
-          path: /var/vcap/data/
+          path: /var/vcap/store/docker/docker/containers
       - name: loggregator-tls-certs
         secret:
-          secretName: <%= p('scf.secrets_generation_name') %>
```

Comment:
We don't have access to this property if Eirini is deployed without SCF. We could create a property in this bosh release to be able to parameterize the secret. I think it's also confusing to reach for a property defined outside the job where it's being used.

Reply:
https://github.com/cloudfoundry-community/eirini-bosh-release/pull/25/files#diff-d08bb26fb12ca89f739e70e59f599243R25

Reply:
@jimmykarily The changes in your PR above are still unfortunately not quite entirely addressing the concerns we have about this errand not creating everything it needs itself. The changes in our PR [here](https://github.com/cloudfoundry-community/eirini-bosh-release/pull/22/files#diff-1ff2454ad97a02bc44853aa5dcde2a90R10) creates the secret that `opi.certs_secret_name` depends on so that we don't have to do things out of band from the errand. Specifically the `opi.certs_secret_name` secret still depends on a particular secret existing in the cluster already before the errand runs. 

We would strongly prefer this approach so the errand can take care of everything for us that needs to be bootstrapped. The approach we've suggested should also work for y'all, just need to provide the values you were getting from that existing secret as properties when you configure the errand. Of course, if that is significantly more difficult for you all, please do let us know and we can try to find something more amenable.